features/step_definitions/hangout_steps.rb

```diff
@@ -182,4 +185,7 @@
   page.should have_field('hangout_url', with: @hangout_url)
 end
 
-
+Then(/^a separate event instance is not created$/) do
+  expect(EventInstance.where("created_at >= ?", Time.now.beginning_of_day).size)
```

Comment:
I worry slightly that this test is dependent on the current time - if the test suite runs over the midnight boundary it could pass for the wrong reason.  Could we hard code the date based on the date set at the beginning of the cuke test?